From 591d2512efc15adb4ca05b66ad152ed2b25e2f4d Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 22 Nov 2019 12:23:26 +0800
Subject: [PATCH] ARM: dts: rockchip: rk3126-bnd-d708: fix clang warning

In file included from arch/arm/boot/dts/rk3126-bnd-d708.dts:7:
arch/arm/boot/dts/rk3126-bnd-d708.dtsi:558:38: warning: '/*' within block comment [-Wcomment]
                                /* <2 RK_PB1 1 &pcfg_pull_none>, /* HSYNC */
                                                                 ^

Change-Id: Id05936c12f3cbad6bcbf9ed5a4eacb149e8bc5d4
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/boot/dts/rk3126-bnd-d708.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/rk3126-bnd-d708.dtsi b/arch/arm/boot/dts/rk3126-bnd-d708.dtsi
index f10be4ba2361..d1545beee14d 100644
--- a/arch/arm/boot/dts/rk3126-bnd-d708.dtsi
+++ b/arch/arm/boot/dts/rk3126-bnd-d708.dtsi
@@ -555,8 +555,8 @@
 			rockchip,pins =
 				/* depend on the hardware */
 				<2 RK_PB0 1 &pcfg_pull_none>, /* DCLK */
-				/* <2 RK_PB1 1 &pcfg_pull_none>, /* HSYNC */
-				/* <2 RK_PB2 1 &pcfg_pull_none>, /* VSYNC */
+				/* <2 RK_PB1 1 &pcfg_pull_none>, *//* HSYNC */
+				/* <2 RK_PB2 1 &pcfg_pull_none>, *//* VSYNC */
 				<2 RK_PB3 1 &pcfg_pull_none>, /* DEN */
 				<2 RK_PB4 1 &pcfg_pull_none>, /* DATA10 */
 				<2 RK_PB5 1 &pcfg_pull_none>, /* DATA11 */
@@ -566,12 +566,12 @@
 				<2 RK_PC1 1 &pcfg_pull_none>, /* DATA15 */
 				<2 RK_PC2 1 &pcfg_pull_none>, /* DATA16 */
 				<2 RK_PC3 1 &pcfg_pull_none>; /* DATA17 */
-				/* <2 RK_PC4 1 &pcfg_pull_none>, /* DATA18 */
-				/* <2 RK_PC5 1 &pcfg_pull_none>, /* DATA19 */
-				/* <2 RK_PC6 1 &pcfg_pull_none>, /* DATA20 */
-				/* <2 RK_PC7 1 &pcfg_pull_none>, /* DATA21 */
-				/* <2 RK_PD0 1 &pcfg_pull_none>, /* DATA22 */
-				/* <2 RK_PD1 1 &pcfg_pull_none>; /* DATA23 */
+				/* <2 RK_PC4 1 &pcfg_pull_none>, *//* DATA18 */
+				/* <2 RK_PC5 1 &pcfg_pull_none>, *//* DATA19 */
+				/* <2 RK_PC6 1 &pcfg_pull_none>, *//* DATA20 */
+				/* <2 RK_PC7 1 &pcfg_pull_none>, *//* DATA21 */
+				/* <2 RK_PD0 1 &pcfg_pull_none>, *//* DATA22 */
+				/* <2 RK_PD1 1 &pcfg_pull_none>; *//* DATA23 */
 		};
 	};
 
-- 
2.35.3

